aptos-move/aptos-vm/src/move_vm_ext/respawned_session.rs

```diff
@@ -155,6 +157,36 @@ impl<'r> TResourceView for ExecutorViewWithChangeSet<'r> {
     }
 }
 
+impl<'r> TResourceGroupView for ExecutorViewWithChangeSet<'r> {
+    type Key = StateKey;
+    type Layout = MoveTypeLayout;
+    type Tag = StructTag;
+
+    fn resource_group_size(&self, _state_key: &Self::Key) -> anyhow::Result<u64> {
+        // In respawned session, gas is irrelevant, so we return 0 (GroupSizeKind::None).
+        Ok(0)
+    }
+
+    fn get_resource_from_group(
+        &self,
+        state_key: &Self::Key,
+        resource_tag: &Self::Tag,
+        maybe_layout: Option<&Self::Layout>,
+    ) -> anyhow::Result<Option<Bytes>> {
+        self.change_set
+            .resource_group_write_set()
+            .get(state_key)
+            .and_then(|g| g.inner_ops.get(resource_tag))
+            .map(|op| op.extract_raw_bytes())
+            .ok_or(anyhow::Error::msg("Must be ignored immediately after"))
+            .or_else(|_| {
+                // Not found in change-set, fall back to the base executor view.
+                self.base
+                    .get_resource_from_group(state_key, resource_tag, maybe_layout)
+            })
```

Comment:
(TBH some `?`s and `if-else`s or `match`es will be way more easier to read for us dumb people. the "Must be ignored immediately after" thingy is not really useful to my intellectual level :P )

Reply:
LoL, yes, we fixed this in the base PR, just haven't rebased